### PR TITLE
Update PureVPN configs

### PR DESCRIPTION
**Pull request by ryanlauterbach**
* Update openvpn files for purevpn with latest files as of 22/01/2018
* Add .dockerignore, no point sending .git files to docker context

### CODE REVIEW

File: openvpn/purevpn/Latvia-tcp.ovpn [hunk 1]
```diff
@@ -14,6 +14,6 @@ route-method exe
 route-delay 2
 route 0.0.0.0 0.0.0.0
 float
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass /config/openvpn-credentials.txt /config/openvpn-credentials.txt
```
Comment (haugene) [inline openvpn/purevpn/Latvia-tcp.ovpn]: Something has happened here...

File: openvpn/purevpn/Latvia-udp.ovpn [hunk 2]
```diff
@@ -13,7 +13,7 @@ mute 20
 route-method exe
 route-delay 2
 route 0.0.0.0 0.0.0.0
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass /config/openvpn-credentials.txt /config/openvpn-credentials.txt
```
Comment (haugene) [inline openvpn/purevpn/Latvia-udp.ovpn]: And here

File: openvpn/purevpn/Luxembourg.St-tcp.ovpn [hunk 3]
```diff
@@ -14,6 +14,6 @@ route-method exe
 route-delay 2
 route 0.0.0.0 0.0.0.0
 float
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass /config/openvpn-credentials.txt /config/openvpn-credentials.txt
```
Comment (haugene) [inline openvpn/purevpn/Luxembourg.St-tcp.ovpn]: ++

File: openvpn/start.sh [hunk 4]
```diff
@@ -21,7 +21,7 @@ then
 		OPENVPN_CONFIG=$vpn_provider_configs/default.ovpn
 	fi
 else
-	echo "No VPN configuration provided. Using default."
+	echo "No VPN configuration provided $OPENVPN_CONFIG. Using default."
```
Comment (haugene) [inline openvpn/start.sh]: This will only happen when OPENVPN_CONFIG is empty, why append it?

File: openvpn/start.sh [hunk 5]
```diff
@@ -81,4 +81,5 @@ if [ -n "${LOCAL_NETWORK-}" ]; then
   fi
 fi
 
+echo "Exec: openvpn $TRANSMISSION_CONTROL_OPTS $OPENVPN_OPTS --config '$OPENVPN_CONFIG'"
```
Comment (haugene) [inline openvpn/start.sh]: Meant for debug purposes?

**Comment by haugene** [comment]
Hi! First of all thanks for contributing, looking forward to merging updated configs. 

I did find 10 files (quick search) where I think something has gone wrong with sed appending. Commented on a few of them. Also wondering if the changes to openvpn/start.sh was intentional.

Please update the PR and I'll merge it :)

**Comment by haugene** [comment]
Hi. Thanks for updating the PR, the ovpn files looks good and ready to merge. Still one remaining change in openvpn/start.sh that I'm wondering if is intentional?

**Comment by ryanlauterbach** [comment]
Debug line removed, thanks.

**Comment by haugene** [comment]
Push changes?

**Comment by ryanlauterbach** [comment]
oops pushed

**Comment by haugene** [comment]
There it is. But you still have changes committed in openvpn/start.sh. You've removed two other log lines? Fix that and maybe squash your commits if you could?

**Comment by haugene** [comment]
Sweet 👍

**Pull request merged**